ydb/library/yql/udfs/common/datetime2/test_bigdates/ya.make

```diff
@@ -0,0 +1,13 @@
+YQL_UDF_YDB_TEST()
+
+DEPENDS(ydb/library/yql/udfs/common/datetime2)
+
+TIMEOUT(300)
+
+SIZE(MEDIUM)
+
+IF (SANITIZER_TYPE == "memory")
+    TAG(ya:not_autocheck) # YQL-15385
+ENDIF()
+
+END()
```

Comment:
I guess `YQL_ABI_VERSION` is missing here. See other test suites for UDFs.

Reply:
May be you meen `ydb/library/yql/udfs/common/datetime2/ya.make` ?

Reply:
Yeah, precisely.